adapted-articles/S/Shepherd.xml

```diff
@@ -2,9 +2,9 @@
+<p class="fl">One who took complete care of a flock of sheep. His task was to find grass and water for the sheep, to protect them from wild animals (<a href="?bref=Amos.3.12">Am 3:12</a>), to look for and restore those that got lost (<a href="?bref=Ezek.34.8">Ez 34:8</a>; <a href="?bref=Matt.18.12">Mt 18:12</a>), to lead the flock out of the fold each day, and to return the flock to the fold at the close of the day (<a href="?bref=John.10.2-4">Jn 10:2-4</a>).</p>
+<p>The figure of the shepherd and his sheep is important in the NT. Jesus is the Good Shepherd who gives his life for the sheep (<a href="?bref=Matt.18.10-14">Mt 18:10-14</a>; <a href="?bref=Mark.6.34">Mk 6:34</a>; <a href="?bref=John.10.1-42">Jn 10</a>; <a href="?bref=Heb.13.20">Heb 13:20</a>). The analogy of the shepherd and the flock is richly used in <a href="?bref=Ps.23.1-6">Psalm 23</a>, <a href="?bref=Ezek.34.1-31">Ezekiel 34</a>, and <a href="?bref=John.10.1-42">John 10</a>. God was the Shepherd of Israel (<a href="?bref=Gen.49.24">Gn 49:24</a>; <a href="?bref=Ps.23.1">Pss 23:1</a>; <a href="?bref=Ps.80.1">80:1</a>; <a href="?bref=Isa.40.11">Is 40:11</a>). When unfaithful shepherds failed Israel, God intervened and appointed his servant David as a faithful shepherd over them (<a href="?bref=Ezek.34.11-16,23-24">Ez 34:11-16, 23-24</a>).</p>
+<p>The NT imagery comes from an OT and Palestinian background. In the Jewish economy, the shepherd who was taking care of a flock of sheep or goats held a responsible position. Great flocks had to be moved from place to place, and it was necessary to protect them from wild animals and robbers. Because of the fundamental role of shepherds in the ancient world, the word “shepherd” became a common term for a ruler. The kings of Assyria, Babylon, and Egypt were often referred to as shepherds who protected their people. This imagery formed the background for the OT, where the same usage is found. God is pictured as the shepherd of Israel, concerned for every aspect of his people’s welfare. Rulers and leaders of the people are often referred to as shepherds (<a href="?bref=Num.27.17">Nm 27:17</a>; <a href="?bref=1Kgs.22.17">1 Kgs 22:17</a>; <a href="?bref=Jer.10.21">Jer 10:21</a>; <a href="?bref=Jer.12.10">12:10</a>; <a href="?bref=Jer.22.22">22:22</a>; <a href="?bref=Jer.23.1-2">23:1-2</a>).</p>
```

Comment:
the shepherd who was taking care of a flock of sheep --> the shepherd who took care of a flock of sheep